zoneminder/zoneminder-event-cleanup.sh

```diff
@@ -4,23 +4,31 @@
 # Purpose:	Clean up old Zoneminder events but exclude archived events
 # Authors:	Claudio Kuenzler (2018)
 # Doc:		https://www.claudiokuenzler.com/blog/814/how-to-manually-clean-up-delete-zoneminder-events
-# History:	
-# 2018-12-14 First version
+# History:
+# 2020-08-06 changed for zoneminder 1.24+ (@Brawn1)
+# 2020-07-24 added Docker Mysql (@Brawn1)
 # 2018-12-18 DB Name variable in first query
+# 2018-12-14 First version
 ###########################################################################
 # User variables
-olderthan=2 # Defines the minimum age in days of the events to be deleted
-zmcache=/var/cache/zoneminder/events # Defines the path where zm stores events
+olderthan=7 # Defines the minimum age in days of the events to be deleted
```

Comment:
We should add a parameter to the script. I'm fine with a default of 7 days, but user should have the option to easily use another limit.